dms_auto_classification/models/dms_classification_template.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class DmsClassificationTemplate(models.Model):
+    _name = "dms.classification.template"
+    _order = "sequence, name"
```

Comment:
Always put `id` at the end

```suggestion
    _order = "sequence,name,id"
```